packages/lens/lens.1.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "lens"
+version: "1.0.1"
+ocaml-version: >= "4.01.0"
+maintainer: "Paolo Donadeo <p.donadeo@gmail.com>"
+authors: [ "Alessandro Strada <alessandro.strada@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/pdonadeo/ocaml-lens"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "lens"]
+]
+depends: [
+  "ocamlfind"
```

Comment:
You may want to use [oasis2opam](https://github.com/ocaml/oasis2opam) which will do that — and more — for you (using the Git version at the moment but I could make a release).  In order to have the (new version) of `ocaml-version: >= "4.01.0"`, you need to add that constraint to `_oasis`.